<?xml version="1.0"?>
<launch>
    <arg name="sound_collection" default="willow-sound" />
    <node pkg="ropod_sound_communication" type="sound_communication" respawn="false" name="ropod_sound_communication" output="screen">
       <param name="sound_config" value="$(find ropod_sound_communication)/config/sound_config.yaml"/>
       <param name="sound_collection" value="$(find ropod_sound_collections)/collections/$(arg sound_collection)"/>
       <param name="sound_request_topic" value="/ropod/state_info"/>
       <param name="go_to_progress_topic" value="/ropod/goto/feedback"/>
       <param name="dock_progress_topic" value="/collect_cart/feedback"/>
    </node>
</launch>
